test/msc3916_downloads_suite_test.go

```diff
@@ -85,14 +86,16 @@ func (s *MSC3916DownloadsSuite) TestClientDownloads() {
 	assert.NoError(t, err)
 	assert.Equal(t, http.StatusUnauthorized, raw.StatusCode)
 
-	raw, err = client1.DoRaw("GET", fmt.Sprintf("/_matrix/client/v1/media/download/%s/%s", origin, mediaId), nil, "", nil)
-	assert.NoError(t, err)
-	assert.Equal(t, http.StatusOK, raw.StatusCode)
-	test_internals.AssertIsTestImage(t, raw.Body)
-	raw, err = client1.DoRaw("GET", fmt.Sprintf("/_matrix/client/v1/media/download/%s/%s/whatever.png", origin, mediaId), nil, "", nil)
-	assert.NoError(t, err)
-	assert.Equal(t, http.StatusOK, raw.StatusCode)
-	test_internals.AssertIsTestImage(t, raw.Body)
+	for _, authedClient := range []*test_internals.MatrixClient{client1, clientGuest} {
```

Comment:
likewise here - code duplication (or an otherwise useful stacktrace) is preferred

Reply:
50b37b2